src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml

```diff
@@ -1,13 +1,13 @@
 <zapaddon>
 	<name>Passive scanner rules (alpha)</name>
-	<version>10</version>
+	<version>11</version>
```

Comment:
It's still a manual process, not sure if should be that automatic ;)
Simple steps, build the add-on, upload it and update the ZapVersions file in zap-admin repo. Usually @psiinon or myself.